src/applications/ezr/tests/unit/components/IntroductionPage/SaveInProgressInfo.unit.spec.js

```diff
@@ -183,6 +185,51 @@ describe('ezr <SaveInProgressInfo>', () => {
           expect(selector).to.contain.text(content['alert-server-title']);
         });
       });
+
+      context('when the user has a preferred facility on file', () => {
+        it('should not render preferred facility alert', () => {
+          const { props, mockStore } = getData({
+            showLoader: false,
+            loggedIn: true,
+            loaState: 3,
+            enrollmentStatus: 'enrolled',
+          });
+          const { container } = render(
+            <Provider store={mockStore}>
+              <SaveInProgressInfo {...props} />
+            </Provider>,
+          );
+          const selector = container.querySelector(
+            '[data-testid="ezr-preferred-facility-alert"]',
+          );
+          expect(selector).to.not.exist;
+        });
+      });
+
+      context(
+        'when the user does not have a preferred facility on file',
+        () => {
+          it('should render preferred facility alert', () => {
+            const { props, mockStore } = getData({
+              showLoader: false,
+              loggedIn: true,
+              loaState: 3,
```

Comment:
I also see it here which makes my believe it was intentional but I just want to confirm and maybe hear the reasoning for the naming convention for my reference.